### ISSUE TITLE
Is there a way to edit multiple point clouds together in one go? 

### ISSUE THREAD

**Issue by kathanvyas** [cnr-isti-vclab/meshlab#800]
I have imported 50 point clouds and all fo them are aligned. I need to remove some parts of all point clouds which are also aligned. is there some way to do it for all point clouds together? Currently I open one, edit, save and move to next.

**Comment by alemuntoni** [comment]
Do you mean to make a selection of all the layers and not only the selected one? I think that's not possible right now.
The only way would be to merge all the layers. I'll set this as a feature request.

**Comment by kathanvyas** [comment]
@alemuntoni yes so think about a scenario where i have 100 meshes. I need to remove the same region (lets assume background) from all those 100 meshes. its basically the same area and right now what I have to do is go to each of them individually and remove the area. and then save it. even though the area to be removed is the same (by same I mean same x-yz cordinates). Is there a way I can make changes in all 100 of them together?
